aspnetcore/security/data-protection/introduction.md

```diff
@@ -18,9 +18,18 @@ Web apps often need to store sensitive data. The Windows data protection API ([D
 
 The ASP.NET Core data protection stack was designed to:
 
-* Provide a built in solution for most Web scenarios.
-* Address many of the deficiencies of the previous encryption system. 
-* Serve as the replacement for the `<machineKey>` element in ASP.NET 1.x - 4.x.
+* Provide a built in solution for most Web scenarios, including web farms and Azure Container Apps.
+* Address many of the deficiencies of the previous encryption system.
+
+## Key consistency in web farms and Azure Container Apps
+
+If the developer has not configured data protection in ASP.NET Core, the environment variable `autoConfigureDataProtection` is set to `true`.  When `autoConfigureDataProtection` is set to `true`, the data protection keys are read only. Setting the data configuration keys to read only is designed to maintain data consistency in web farms and [Azure Container Apps](/azure/container-apps/overview).
+
+The `ReadOnlyDataProtectionKeyDirectory`, `DOTNET_ReadOnlyDataProtectionKeyDirectory` and `DataProtectionKeyDirectory` configuration values:
```

Comment:
Somewhere along the line, a typo appears to have been introduced.  There's only one environment variable name, `ReadOnlyDataProtectionKeyDirectory`, and it can be used as-is or with a `DOTNET_` or `ASPNETCORE_` prefix.  The precedence order of the various prefixes is common to all configuration properties.